### PR TITLE
chore(packagers): yarn workspaces pnp

### PR DESCRIPTION
**Pull request by millsp**

### CODE REVIEW

**Comment by Jolg42** [comment]
Note:
fails with 
```
Run node .github/scripts/check-folders.js
There are some folders in e2e-tests that are not used in test.yaml or optional-test.yaml workflows
[
  'packagers/yarn3-workspaces-pnp/packages/sub-project-1',
  'packagers/yarn3-workspaces-pnp/packages/sub-project-2'
]
Error: Process completed with exit code 1.
```

You might need to ignore these from https://github.com/prisma/e2e-tests/blob/dev/.github/scripts/check-folders.js

File: .github/scripts/check-folders.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ async function main() {
         'platforms-serverless/firebase-functions/functions', // Firebase root doesn't have package.json but is included
         'generic/basic', // generic/basic doesn't use Github action matrix feature which we parse to find out the differences
         'packagers/yarn-workspaces/prisma-project', // Yarn workspaces root doesn't have package.json but is included
+        'packagers/yarn3-workspaces-pnp/sub-project-1',
+        'packagers/yarn3-workspaces-pnp/sub-project-2',
```
Comment (Jolg42) [inline .github/scripts/check-folders.js]: Looks like you could change to
```suggestion
        'packagers/yarn3-workspaces-pnp/packages',
```

Comment (millsp) [reply .github/scripts/check-folders.js]: Ah thanks, I thought I was going mad.

**Comment by Jolg42** [comment]
So if I understand it right [this failure](https://github.com/prisma/e2e-tests/runs/5317864134?check_suite_focus=true#step:6:471) is expected
```
+ cd ../../packages/sub-project-2
+ sh test.sh
Error: ENOENT: no such file or directory, open '/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/packages/sub-project-2/src/schema.prisma'
    at Object.openSync (fs.js:462:3)
    at NodeFS.openSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:2761:24)
    at makeCallSync.subPath.subPath (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:4741:26)
    at ZipOpenFS.makeCallSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:5306:14)
    at ZipOpenFS.openSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:4740:17)
    at VirtualFS.openSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:4355:24)
    at PosixFS.openSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:4355:24)
    at URLFS.openSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:4355:24)
    at Object.readFileSync (fs.js:364:35)
    at NodeFS.readFileSync (/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/.pnp.cjs:3040:24) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/home/runner/work/e2e-tests/e2e-tests/packagers/yarn3-workspaces-pnp/packages/sub-project-2/src/schema.prisma',
  clientVersion: '3.11.0-dev.5'
}
```
 and will be fixed with https://github.com/prisma/prisma/pull/12000

**Review by Jolg42 — APPROVED** [review]
🥳

**Pull request merged**